codalab/bundles/runnable_bundle.py

```diff
@@ -0,0 +1,40 @@
+'''
+RunnableBundle is an abstract Bundle supertype for bundles that need to be
```

Comment:
I like the refactoring, but this name is confusing (that runnable includes make and run).  Let's call it DerivedBundle.